### ISSUE TITLE
Ability to edit/update game model

### ISSUE THREAD

**Issue by terryalexis** [Brich96/usu-gdsf#42]

**Comment by hagen-larsen-hl** [comment]
Is this issue to update the game model to reflect the structure they are using in the front end?

**Comment by terryalexis** [comment]
@hagen-larsen-hl  No, this is the issue to add the ability to update a specific game's info in the database.  This one is held up by that task to make the game model the same as what's used on the front-end.

**Comment by terryalexis** [comment]
@hagen-larsen-hl I'm not sure which task is the one that's fixing up the model...

**Comment by terryalexis** [comment]
Duplicate of one that was just completed
